guide/getting-started-with-testcontainers-for-python/index.adoc

```diff
@@ -44,13 +44,13 @@ $ source venv/bin/activate
 
 We are going to use https://www.psycopg.org/psycopg3/[psycopg3] for talking to the Postgres database,
 https://pytest.org/[pytest] for testing,
-and https://testcontainers-python.readthedocs.io/en/latest/README.html[testcontainers-python] for running a PostgreSQL database in a container.
+and https://testcontainers-python.readthedocs.io/[testcontainers-python] for running a PostgreSQL database in a container.
```

Comment:
@sivaprasadreddy - we had to rename this one because pypi started rejecting the readme :cry: so we split it up and its now "index.rst" not sure if this is super unorthodox or what but linking to the main site seems to work still